javers-core/src/test/groovy/org/javers/core/JaversRepositoryE2ETest.groovy

```diff
@@ -649,4 +649,36 @@ class JaversRepositoryE2ETest extends Specification {
       snapshots[1].version == 1
 
     }
+
+    def "should query for Entity snapshot with given commit id in a form of CommitId"() {
```

Comment:
wycofuję się z poprzedniej uwagi. Jak to trochę nad tym popracowałem to wyszło całkiem zgrabnie :)